aries-core/src/js/components/core/Tile/Tile.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Box, Header, Heading } from 'grommet';
+
+const PAD_SIZES = ['xxsmall', 'xsmall', 'small', 'medium', 'large', 'xlarge'];
+
+export const Tile = ({ children, heading, pad, ...rest }) => {
+  const tilePad =
+    pad === true ? { horizontal: 'small', top: 'small', bottom: 'small' } : pad;
+
+  return (
+    <Box elevation="medium" round="small" {...rest}>
```

Comment:
Because the Tile is setting `round="small"`, I think it should also set `overflow="hidden"` to prevent children from exceeding the rounded corners.

Reply:
Done. Thanks for elaborating.